app/helpers/report_dashboard_helper.rb

```diff
@@ -1,4 +1,4 @@
-module ScaptimonyReportDashboardHelper
+module ReportDashboardHelper
```

Comment:
Maybe rename this to `ArfReportDashboardHelper` so there won't be confusion with Foreman's `Report`